presto-test-jdbc-compatibility-old-driver/src/test/java/io/prestosql/TestJdbcResultSetCompatibilityOldDriver.java

```diff
@@ -32,6 +32,8 @@
 import static io.prestosql.TestingServerUtils.setTestingServer;
 import static org.assertj.core.api.Assertions.assertThat;
 
+// We need to keep @Test(singleThreaded = true) even though current version of superclass does not require it.
+// This class is compiled against older versions of TestJdbcResultSet which require single threaded execution of tests.
```

Comment:
either remove `forceTestNgToRespectSingleThreaded` or add the (now deleted) comment on it.
otherwise that method triggers obvious question "how does it help" without providing the answer.